actin/local/serve/pull_serve_resources_into_local_actin_resource_repo

```diff
@@ -7,5 +7,5 @@ local_base_path="${HOME}/hmf/repos/actin-resources-private"
 
 info "Pulling SERVE resources from ${remote_base_path} into ${local_base_path}"
 
-gcloud compute --project hmf-crunch scp ${remote_base_path}/*37*tsv ${local_base_path}/serve_db/
-gcloud compute --project hmf-crunch scp ${remote_base_path}/*38*tsv ${local_base_path}/serve_db_38/
\ No newline at end of file
+gcloud compute --project hmf-crunch scp ${remote_base_path}/37/*37*tsv ${local_base_path}/serve_db/37/
+gcloud compute --project hmf-crunch scp ${remote_base_path}/38/*38*tsv ${local_base_path}/serve_db/38/
```

Comment:
This won't work I think, as SERVE (the `remote_base_path` writes everything into a single output dir.

I think you should only have to change the `local_base_path`

Reply:
Good one, thanks